README.md

```diff
@@ -286,6 +304,16 @@ Because of the great properties of OCaml 5 memory model (see the
 more details), not a lot can go wrong here. At least, data corruption or
 segmentation fault won't happen like it can in other languages.
 
+## Safe and unsafe data structures
+Some data structures are available in two versions: a normal version and a more optimized but **unsafe** version. The **unsafe** version utilizes `Obj.magic` in a way that may be unsafe with `flambda2` optimizations.
```

Comment:
Let's format .md files with prettier.  I recommend setting up your editor so that it does that automatically.

Reply:
It should have been working this way, but apparently, my editor config was not working right ! Thanks for catching it.